exam/backend/controllers/create_exam_controller.js

```diff
@@ -0,0 +1,30 @@
+const db = require('../models') ;
+const sequelize = require('sequelize') ;
+const jwt = require('jsonwebtoken');
+const utilities = require('../helpers/utilities');
+
+exports.createExam = async ( req , res  ) =>{
+    let new_agency_id = req.body.agency_id ;
+    let new_name = req.body.name ;
+    let new_details = req.body.details ;
+    let new_max_marks = req.body.max_marks ;    
+    let new_time = req.body.time ;
+    let new_avg_marks = req.body.avg_marks ;
```

Comment:
How can you get average exam as in input to create exam?? It will be 0 to begin with and then will be calculated every time a student gives a test paper associated to this exam ID.

Reply:
For that we will have to count the number of users attempted that exam and will have to be done afterwards but why do we need average marks for an exam ? We need average marks for a paper as different papers can have different maximum marks scale so instead of defining here we can define average marks in mock papers.

Reply:
Let us take this up in today's call.

Reply:
Okay I get it that it is assumed that every paper of exam will have same format but still as toughness may vary it will be a more logical to give paper average marks instead of the exam.
Like JEE Mains 2015 had average marks 100 , JEE Mains 2016 had average marks 90 etc etc.